src/schema/alerts.ts

```diff
@@ -146,10 +147,22 @@ export const updateAlerts = async <T = GQLUpdateAlertsInput>(
   }
 
   if (!alerts) {
-    return saveReturnAlerts(await repo.save({ userId, ...data }));
+    return saveReturnAlerts(
+      await repo.save({
+        userId,
+        ...data,
+        flags: { ...flags, lastReferralReminder: new Date() },
```

Comment:
Is this now duplicate, we already pass it as `flags` from `updateLastReferralReminder`?

Reply:
Changed it in last commit :)